### DISCUSSION TITLE
リポジトリにおけるRTDファイルの階層について

### DISCUSSION THREAD

**Discussion by mmer547** [FEABeRjp/FEABeR#16] [General]
RTDのページファイルは現在リポジトリトップにおいていますが、CoFEAプロジェクトと同様にディレクトリで分けるべきでしょうか?ご意見伺いたく願います。

**Comment by tkoyama010** [comment]
現状特に困っていないので as is でよいかと思います。問題が発生したら再度議論するでいかがでしょう。
